web/src/components/Plugin/UI/plugin.tsx

```diff
@@ -27,6 +27,11 @@ import ProxyMirror from './proxy-mirror';
 import LimitConn from './limit-conn';
 import RefererRestriction from './referer-restriction';
 import Cors from './cors';
+<<<<<<< HEAD
+=======
+import RequestId from './request-id';
+import IpRestriction from './ip-restriction';
+>>>>>>> 18ade18 (feat: ip-restriction added to plugin list)
```

Comment:
Are there conflicting files that have not been dealt with yet?

Reply:
Yes this PR is in a draft mode right now, it may have many errors